### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by benlooi**
Filled up details and basic explanation of how it works. Did trial and error, so I am not certain if we need to install tessel at the folder /node_modules or /node_modules/node-red  . I did both anyway.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,58 @@
 # tessel-node-red
 A library that allows users to get Node Red working with Tessel and it's module ecosystem.
```
Comment (Frijol) [inline README.md:2]: should be 'its'

File: README.md [hunk 2]
```diff
@@ -1,26 +1,58 @@
...
+As I am quite a noob, here are the current issues :
+The JS and HTML files as of today are not working, i.e, you can see the nodes and dialog boxes ( I adapted heavily from the Arduino Firmata sample), so I need experts out there to help get it right.
 
 # Examples
 Here is some example setup/pics/code etc to get a basic example running with Node Red.
 
 # Contact
-Having an issue with any of this? Check out [the forumns](https://forums.tessel.io/) and ask for help!
\ No newline at end of file
+Having an issue with any of this? Check out [the forumns](https://forums.tessel.io/) and ask for help!
```
Comment (johnnyman727) [inline README.md]: Misspelling: `forumns` -> `forums`

File: README.md [hunk 3]
```diff
@@ -1,26 +1,58 @@
...
+
+#Setup
+Once installed, you can start the node-red server with node-red and point your browser to http://localhost:1880.
+If it is showing up in your browser, congrats! Press Ctrl-C to stop node-red. We'll come back to it with tessel nodes....be patient.
+
+For Tessel setup, go to the folder where node-red is installed. It should be found in your /node_modules/ folder in your system.
+cd node_modules
+
+Inside this folder, install tessel.
+npm install -g tessel
```
Comment (johnnyman727) [inline README.md]: Should this be a global install?

Comment (benlooi) [inline README.md]: I'm not sure...haven't been able to get it to work yet. Maybe a better bet would be a local install in the node_modules folder.

**Pull request merged**